sdk/src/main/java/com/ifountain/opsgenie/client/model/contact/AddContactRequest.java

```diff
@@ -1,24 +1,39 @@
 package com.ifountain.opsgenie.client.model.contact;
 
-import com.ifountain.opsgenie.client.model.BaseUserRequest;
+import com.ifountain.opsgenie.client.OpsGenieClientConstants;
+import com.ifountain.opsgenie.client.OpsGenieClientValidationException;
 import com.ifountain.opsgenie.client.model.beans.Contact;
 import com.ifountain.opsgenie.client.model.beans.Contact.Method;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.Objects;
 
 /**
  * Container for the parameters to make an add contact api call.
  *
  * @author Mehmet Mustafa Demir
  * @see com.ifountain.opsgenie.client.IContactOpsGenieClient#addContact(AddContactRequest)
  */
-public class AddContactRequest extends BaseUserRequest<AddContactResponse, AddContactRequest> {
+public class AddContactRequest extends BaseContactRequest<AddContactResponse, AddContactRequest> {
```

Comment:
Did we finalise editing current request or we were thinking of create AddContectRequestV2. @trabab ?

Reply:
@prakhar-mudaiya , we are checking if the v1 versions are working, if so then we need to create the separate implementation for the v2 versions keeping in mind the backward compatibility of the apis.